//package com.sincetimes.redwar.game.comnunicate.protocol;


message RseUpdateMissions
{
	optional int32 ret = 1;
}